src/constants/endpoints.ts

```diff
@@ -1,19 +1,12 @@
 import { Article } from './data'
 
-export interface ProfessionType {
+export interface DisciplineType {
```

Comment:
Do we have a glossary somewhere for these type of terms? I am really confused all the time what is a document, word, discipline, profession and so on. This is not an app specific topic though, but a project wide. Should I mention this in ~lunes-general?

Reply:
Yes you are right, it is really confusing and we were talking about this already in lunes calls, but nothing already.
Currently the api has the endpoints:
- disciplines -> trainings_sets -> documents (wich have words)
In the app we have:
- profession -> subprofessions -> documents
Non-technical used naming:
- Bereichen -> Module

As with this PR in the frontend we merge disiplines and trainingssets into one type. I created a ticket and suggested the name "module" for it, but not really happy with it tough, so feed free to suggest an other name for disciplines/trainingssets (actually these are just like "folders" where the documents are inside)
https://issues.tuerantuer.org/browse/LUN-142

But feel also free to open a discussion in ~lunes-general

Reply:
Okay, thanks for explaining :+1: And `subCategories` in `endpoints.ts` is just a legacy of the old structure and basically the same as a training set, i.e. a module with documents as children? Will open a discussion :)

Reply:
Maybe just only using the term profession/disclipine would also work instead of renaming to module. That would probably make the purpose a little bit clearer.

Reply:
oke, i will adjust the ticket so it will be renamed to discipline.

Reply:
I like that, that way we have nearly the same wording as the api as well :)